virttest/virt_vm.py

```diff
@@ -1209,6 +1212,8 @@ def print_guest_network_info():
                                                  restart_network, username,
                                                  password, False)
             if serial:
+                if set_dumb_terminal:
+                    session.cmd("export TERM=dumb")
```

Comment:
VMs can be Windows guests therefore a check of OS type is required before setting the `TERM` variable.
```python
if set_dumb_terminal and self.params.get("os_type") != "windows":
    session.cmd("export TERM=dumb")
```
https://github.com/avocado-framework/avocado-vt/blob/d4c323aac396667cde94a3d22ba419647c8e6751/virttest/shared/cfg/guest-os/Windows.cfg#L6